api/modules/publishedProjects/model.js

```diff
@@ -0,0 +1,31 @@
+var BaseModel = require('../../classes/base_model');
+
+var Projects = require('../projects/model');
+
+var instanceProps = {
+  tableName: 'publishedProjects',
+  project: function() {
+    return this.belongsTo(Projects);
```

Comment:
This uses bookshelf to create more abstract relationships as a layer on top of the DB's formal schemas. This method creates a query that essentially says, *get me the `project` whose `published_id` value is this `publishedProjects`' `id`*